<launch>
	<arg name="veh" doc="Name of vehicle. ex: megaman"/>
	<arg name="local" default="false" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
	<arg name="config" default="baseline" doc="Specify a config."/>
	<arg name = "param_file_name" default = "default" />
	<arg name = "pkg_name" value = "gd_motion" />
	<arg name = "node_name" default = "gd_lane_filter_node" />

    <remap from="gd_lane_filter_node/car_cmd" to="gd_text_car_cmd_node/car_cmd"/>
    <remap from="gd_lane_filter_node/caffe_predictions" to="gd_L2R_text_text_node/caffe_predictions"/>
	<node ns="$(arg veh)" pkg="gd_motion" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
		<!--rosparam command="load" file="$(find guidedog)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/-->
	</node>
</launch>
